Themis/Views/Exercises/ExerciseView.swift

```diff
@@ -63,6 +74,7 @@ struct ExerciseView: View {
                 await exerciseVM.fetchExerciseStatsForDashboard(exerciseId: exercise.id)
                 await submissionListVM.fetchTutorSubmissions(exerciseId: exercise.id)
             }
+            print(orientation.isLandscape)
```

Comment:
Remove this again

Reply:
> Is this new animation intended? I think it looks a bit strange.
> 
>  screenshot-2023-01-27_000953.mp4 
> Also, maybe we could just stack the diagrams in a `VStack` when in portrait mode and make the scrolling vertical? I believe this would make more sense.
> 
> The code looks very good so far!

Strange, there weren't any sliding animations before, as you can see from the screen recording. I'll look into this.